### ISSUE TITLE
Modifiers state stucks in initial state (no update) with Gnome Shell 3.10

### ISSUE THREAD

**Issue by wachjose88** [sneetsher/keyboard_modifiers_status#2]
Unfortunately it does not work with Gnome Shell 3.10 in Ubuntu 14.04 LTS. I think this would be important because 14.04 will be supported until April 2019.

thanks Josef

**Comment by sneetsher** [comment]
Thank you Josef, I tried many ways to fix this but I couldn't figure out what's wrong. The problem here is with event loop, the extension get the 1st state but never update.

I had wrote a query to gnome-shell mailing list but reply yet . So i posted a new question to stack-overflow, hopefully someone may help me in this. Here, the links:
- [What have been changed in Gnome Shell event loop v3.10 vs v3.14/v3.16](https://mail.gnome.org/archives/gnome-shell-list/2015-September/msg00006.html)
- [Extension event loop in Gnome 3.10 vs 3.14](https://stackoverflow.com/questions/33693413/extension-event-loop-in-gnome-3-10-vs-3-14)
